### DISCUSSION TITLE
How to use the property renderers?

### DISCUSSION THREAD

**Discussion by codler** [remarkjs/remark#977] [Q&A] [answered]
Where can I find documentation on `renderers` property?

**Comment by ChristianMurphy** [comment, marked as answer]
This appears to be referring to `react-markdown`

Renderers were removed two major versions ago in version 6 https://github.com/remarkjs/react-markdown/blob/main/changelog.md#600---2021-04-15
The documentation for version 5.0.3 is the last that included `renderers` and can be found by viewing the readme at that point in time https://github.com/remarkjs/react-markdown/tree/5.0.3
